instrument-repl/src/repl.rs

```diff
@@ -522,6 +526,15 @@ impl Repl {
                     Arg::new("path").required_unless_present("help")
                 )
         )
+        .subcommand(
+            Command::new(".reset")
+                .help_template(SUBCMD_TEMPLATE)
+                .about("Cancel any ongoing jobs and send *RST.")
```

Comment:
Maybe remove the fullstop at the end of the line. Same with other 'about' fields